src/SonarScanner.MSBuild.Common/IO/IDirectoryWrapper.cs

```diff
@@ -46,6 +46,9 @@ public interface IDirectoryWrapper
     /// <inheritdoc cref="Directory.Move(string, string)"/>
     void Move(string sourceDirName, string destDirName);
 
-    /// <inheritdoc cref="Directory.EnumerateDirectories(string, string, SearchOption)"/>
+    /// <inheritdoc cref="Directory.EnumerateFiles(string, string, SearchOption)"/>
     public IEnumerable<string> EnumerateFiles(string path, string searchPattern, SearchOption searchOption);
+
+    /// <inheritdoc cref="DirectoryInfo.EnumerateDirectories(string, SearchOption)"/>
+    public IEnumerable<DirectoryInfo> EnumerateDirectories(string path, string searchPattern, SearchOption searchOption);
```

Comment:
I would do it like this, so we can add `public IEnumerable<string> EnumerateDirectories(string path, string searchPattern, SearchOption searchOption);` later. That way, the API is more consistent. Note: The inheritdoc doesn't work on this method.
```suggestion
    public IEnumerable<DirectoryInfo> EnumerateDirectories(DirectoryInfo path, string searchPattern, SearchOption searchOption);
```

Reply:
Yeah. It works surprisingly well in VS.